docs/resources/stripe_meter.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: "stripe"
+page_title: "Stripe: stripe_meter"
+description: |-
+  The Stripe Meter can be created, modified and configured by this resource.
+---
+
+# stripe_meter
+
+With this resource, you can create a billing meter - [Stripe API billing meter documentation](https://docs.stripe.com/api/billing/meter).
+
+A billing meter is a resource that allows you to track usage of a particular event. For example, you might create a billing meter to track the number of API calls made by a particular user. You can then attach the billing meter to a price and attach the price to a subscription to charge the user for the number of API calls they make.
+
+Related guide: [Usage based billing](https://docs.stripe.com/billing/subscriptions/usage-based)
+
+
+## Example Usage
+
+```hcl
+// meter
+resource "stripe_meter" "sample_meter" {
+  display_name = "A Sample meter"
+  event_name   = "sample_meter"
+
+  customer_mapping {
+    event_payload_key = "stripe_customer_id"
+    type              = "by_id"
+  }
+
+  default_aggregation {
+    formula = "sum"
+  }
+
+  value_settings {
+    event_payload_key = "value"
+  }
+}
+
+
+// tiered price for the product using the meter above
```

Comment:
This can be removed from here as this more relates to the price.  Also the price documentation needs update.

Reply:
you mean the whole `resource "stripe_price" "price"` block? I mostly left it to show how to connect the price and the meter (see `recurring.meter` below) but happy to remove.

Reply:
Yes the whole price resource.